src/rtde/rtde_client.cpp

```diff
@@ -77,6 +80,10 @@ bool RTDEClient::init()
     max_frequency_ = CB3_MAX_FREQUENCY;
   }
 
+  // Cap target frequency, defaulting to maximum
+  if (target_frequency_ <= 0.0 || target_frequency_ > max_frequency_)
+      target_frequency_ = max_frequency_;
```

Comment:
We use braces in all (at least most) places.

```suggestion
      {
        target_frequency_ = max_frequency_;
      }
```

Reply:
OK!